encryption/classic-mceliece/classic-mceliece-kem/impl/fpga.yaml

```diff
@@ -0,0 +1,9 @@
+name: Field Programmable Gate Array
```

Comment:
I wonder if we should not store these as three different implementations (space/time/balanced). Pro: it's a different implementation, Con: it's more like the same implementation compiled with different optimization parameters.
(But in the end it'll depend on the frontend's UI anyways so we can maybe just leave it as is for now and change it later if it's really bad)